enterprise/server/quota/quota_manager.go

```diff
@@ -333,19 +333,30 @@ func (qm *QuotaManager) findBucket(nsName string, key string) Bucket {
 	return ns.defaultBucket
 }
 
-func (qm *QuotaManager) Allow(ctx context.Context, namespace string, quantity int64) (bool, error) {
+func (qm *QuotaManager) Allow(ctx context.Context, namespace string, quantity int64) error {
 	key, err := quota.GetKey(ctx, qm.env)
 	if err != nil {
-		log.Warningf("Failed to get quota key: %s", err)
-		return true, nil
+		log.CtxWarningf(ctx, "failed to get quota key for namespace %s: %s", namespace, err)
```

Comment:
nit: most of our log messages are in `Sentence case` since they are functioning as a standalone sentence, vs `status.*Error` messages which are `lowercase` since they can be wrapped like `caller a: caller b: root cause` etc. and then when they are finally logged, they look nice, like: `Failed to do X: caller a: caller b: root cause`

```suggestion
		log.CtxWarningf(ctx, "Failed to get quota key for namespace %s: %s", namespace, err)
```